exalearn-ui/src/app/components/new-grammar/new-grammar.component.html

```diff
@@ -16,20 +16,31 @@
 
 		<div class="question-container">
 			<mat-form-field appearance="fill">
-				<mat-label>Question</mat-label>
+				<mat-label>{{ 'CREATE_PAGE.QUESTION' | translate }}</mat-label>
 				<textarea matInput formControlName="question" placeholder="Ex. What is the capital of GB?"></textarea>
```

Comment:
Can we translate it too? (I'm about placeholder)

Reply:
ok, i'll investigate it